src/zenml/materializers/__init__.py

```diff
@@ -29,8 +29,6 @@
 from zenml.materializers.structured_string_materializer import (
     StructuredStringMaterializer,
 )
-from zenml.materializers.numpy_materializer import NumpyMaterializer
```

Comment:
We need to keep both those files around for backwards compatibility with older artifacts.

Reply:
I know, but we have to be careful keeping the Materializers in this `__init__` file because it is one of the things that we import all the time and the integrations might not be installed. We need to find a graceful way to handle this.

Reply:
Doesn't have to be in the `__init__` I think, just the `pandas/numpy_materializer.py` files

Reply:
Since the comment was on the `__init__`, I wanted to mention it just in case.